include/irr/builtin/glsl/bxdf/common.glsl

```diff
@@ -68,6 +68,12 @@ irr_glsl_LightSample irr_glsl_createLightSampleTangentSpace(in vec3 tangentSpace
     return s;
 }
 
+irr_glsl_LightSample irr_glsl_createLightSampleTangentSpace(in vec3 tangentSpaceV, in vec3 tangentSpaceH, in float VdotH, in mat3 tangentFrame)
+{
+    vec3 tangentSpaceL = irr_glsl_reflect(tangentSpaceV, tangentSpaceH, VdotH);
+    return irr_glsl_createLightSampleTangentSpace(tangentSpaceV, tangentSpaceL, tangentFrame);
+}
+
```

Comment:
I'm sure I already had a function like this...

What are you trying to achieve or use this for?

Reply:
not used any more in material compiler,
 but i think it can be left in the GLSL header as is?